pkg/network/event_windows.go

```diff
@@ -45,6 +49,118 @@ const (
 	UDPProtocol = 17
 )
 
+var (
+	ephemeralRanges = map[ConnectionFamily]map[ConnectionType]map[string]uint16{
+		AFINET: map[ConnectionType]map[string]uint16{
+			UDP: map[string]uint16{
+				"lo": 0,
+				"hi": 0,
+			},
+			TCP: map[string]uint16{
+				"lo": 0,
+				"hi": 0,
+			},
+		},
+		AFINET6: map[ConnectionType]map[string]uint16{
+			UDP: map[string]uint16{
+				"lo": 0,
+				"hi": 0,
+			},
+			TCP: map[string]uint16{
+				"lo": 0,
+				"hi": 0,
+			},
+		},
+	}
+)
+
+func init() {
+	var families = [...]ConnectionFamily{AFINET, AFINET6}
+	var protos = [...]ConnectionType{UDP, TCP}
+	for _, f := range families {
+		for _, p := range protos {
+			l, h, err := getEphemeralRange(f, p)
+			if err == nil {
+				ephemeralRanges[f][p]["lo"] = l
+				ephemeralRanges[f][p]["hi"] = h
+			}
+		}
+	}
+}
+func getEphemeralRange(f ConnectionFamily, t ConnectionType) (low, hi uint16, err error) {
```

Comment:
I'm not a huge fan of named return values, unless they are necessary. Is there a reason we need to use named return values here?